Design/Vehicle-Sizing/Pressure Drop Calculator.py

```diff
@@ -0,0 +1,127 @@
+import math
+from scipy.optimize import fsolve 
+
+def barlows(pressure,outer_diameter,stress,fos):
+    t = (pressure*outer_diameter*fos)/(2*stress)
+
+#Thickness values picked from easily purchased 304 stainless steel tubing from onlinemetals.com
+    if t < 0.020:
+        thickness = 0.020
+    elif t > 0.020 and t < 0.028:
+        thickness = 0.028
+    elif t > 0.028 and t < 0.035: #max thickness flaring tool can handle is 0.035 inches, so it is treated as a hard cap
+        thickness = 0.035
+    else:
+        raise ValueError
+    
+    inner_diameter = outer_diameter - 2*thickness
+    area = inner_diameter**2 * math.pi/4
+    
+    return thickness, inner_diameter, area
+
+class lox():
+    def __init__(self):
+        self.density = 0.0412212402 #lb/in^3 - https://en.wikipedia.org/wiki/Liquid_oxygen
+        self.kv = 0.0024336 #kinematic viscosity, in^2/s - https://www.engineeringtoolbox.com/oxygen-O2-dynamic-kinematic-viscosity-temperature-pressure-d_2081.html?vA=-183&degree=C#
+        self.sg = 1.14 #specific gravity @ 20C, 1 atm - http://www.airproducts.com/~/media/files/pdf/company/safetygram-6.pdf
+
+class rp1():
+    def __init__(self):
+        self.density = 0.0292631065 #lb/in^3 - https://en.wikipedia.org/wiki/RP-1
+        self.kv = 0.00333250667 #kinematic viscosity, in^2/s - https://kinetics.nist.gov/RealFuels/macccr/macccr2008/Bruno2.pdf
+        self.sg = 0.82 #specific gravity assuming density of RP-1 is 820 kg/m^3 - https://kinetics.nist.gov/RealFuels/macccr/macccr2008/Bruno2.pdf
+        
+class helium():
+    def __init__(self):
+        self.density = ((pressure*0.068046)*4/(.0821*298))*3.61273e-5 #lb/in^3, ideal gas law SI -> imperial
+        self.kv = (0.00001317059/12)/self.density #kinematic viscosity, in^2/s - https://www.engineeringtoolbox.com/gases-absolute-dynamic-viscosity-d_1888.html 20C
+        self.sg = 0.138 #https://www.engineeringtoolbox.com/specific-gravities-gases-d_334.html
+        
+def fluid_properties(inputfluid):
+    fluid = inputfluid
+    density = fluid.density
+    kv = fluid.kv
+    sg = fluid.sg
+    
+    return density,kv,sg
+
+def pressuredrop(massflow,density,area,kv,pipe_roughness,k_factor,pressure,pipe_length):
+    
+    gravity = 32.17405 #ft/s^2 - https://www.engineeringtoolbox.com/mass-weight-d_589.html
+
+#Based on massflow input, calculate the velocity. Assumed large tank, so only velocity considered is that at point 2
+#Assume massflow used by engine is a constant value along all points outside the tank
+    velocity = massflow / (density*area)
+
+    reynolds = velocity*inner_diameter/(kv) #https://www.engineeringtoolbox.com/reynolds-number-d_237.html - circular pipe
+
+#friction factor calculation for laminar flow
+#https://www.nuclear-power.net/nuclear-engineering/fluid-dynamics/major-head-loss-friction-loss/friction-factor-for-laminar-flow/
+    if reynolds < 2000:
+        f = 64/reynolds
```

Comment:
Be careful doing this. In python2, this would be treated as integer division if 64 is not a float. Simple solution is to add `from __future__ import division` at the top.